aspnetcore/fundamentals/servers/yarp/extensibility-transforms.md

```diff
@@ -76,6 +71,53 @@ This sample requires YARP 1.1, see https://github.com/microsoft/reverse-proxy/pu
 });
 ```
 
+Custom transforms can only modify a request body if one is already present. They can't add a new body to a request that doesn't have one (for example, a POST request without a body or a GET request). If you need to add a body for a specific HTTP method and route, you must do so in [middleware](xref:fundamentals/middleware/index) that runs before YARP, not in a transform.
+
+The following middleware demonstrates how to add a body to a request that doesn't have one:
+
+```csharp
+public class AddRequestBodyMiddleware
+{
+    private readonly RequestDelegate _next;
+
+    public AddRequestBodyMiddleware(RequestDelegate next)
+    {
+        _next = next;
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        // Only modify specific route and method
+        if (context.Request.Method == HttpMethods.Post &&
```

Comment:
```suggestion
        if (context.Request.Method == HttpMethods.Get &&
```
Nit: Makes it a bit more obvious that the original request did not have a body (as opposed to a `Post` that might have had one)